src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -4,6 +4,7 @@ import {useNavigation} from '@react-navigation/core';
 import PropTypes from 'prop-types';
 
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import FastImage from 'react-native-fast-image';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `FastImage` from 'react-native-fast-image' is added but it's not clear if this library has been added to the project dependencies. If not, it should be added to package.json.
```diff
+ import FastImage from 'react-native-fast-image';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->